src/commands/UploadReactNativeCommand.ts

```diff
@@ -121,7 +121,17 @@ const reactNativeProvideOpts = [
 ]
 
 const reactNativeFetchOpts = [
-  { name: 'fetch', type: Boolean },
+  {
+    name: 'fetch',
+    type: Boolean,
+    description: 'enable fetch mode {bold required}',
+  },
+  {
+    name: 'url',
```

Comment:
`--url` seems a bit easy to confuse for `--endpoint` but I'm not sure what else to call this — `--bundler-url`, `--metro-url`, `--fetch-url` ?

Reply:
yeah definitely something more descriptive than `--url` is required. `--bundler-url` seems ideal, except for the fact it's 1 typo away from `--bundle-url`, which is a valid option for the browser upload command.

Reply:
Yeah that was my hesitation to calling it `bundler-url`, but I think it's probably OK as they're separate sub commands